src/Raven.Studio/wwwroot/App/views/notifications/notificationCenter.html

```diff
@@ -1,96 +1,99 @@
-<div class="studio-alerts" data-bind="with: notificationCenter.recentErrors">
-    <div data-bind="with: currentAlert">
-        <div class="alert alert-dismissable fade in"
-             data-bind="attr: { id: id }, css: { 'alert-info': type === 0, 'alert-success': type === 1, 'alert-warning': type === 2, 'alert-danger': type === 3 || type === 4 }, event: { 'closed.bs.alert': $parent.onAlertHidden.bind($parent) }">
-            <button type="button" class="close" data-dismiss="alert" aria-hidden="true">
-                <i class="fa fa-times"></i>
-            </button>
-            <div class="alert-text">
-                <strong data-bind="text: title, attr:{title:title}"></strong>
+<div data-bind="click: notificationCenter.notificationsState.bind(notificationCenter, true), clickBubble: false">
```

Comment:
Do we need click bubble? I think we need to return false from handler function if bubble is needed, but it shouldn't